plugins/manager/lib/yform/manager/table_api.php

```diff
@@ -67,6 +67,7 @@ public static function setTable(array $table, array $table_fields = array())
             $table_update->update();
 
         }
+        rex_yform_manager_table::deleteCache();
```

Comment:
ich habs durchdebugged und habe immer eine Error-Exception erhalten da beim Import die neu erstellte Tabelle in self::$cache mit Null gesetzt ist und daraufhin die methode ->getFields in Zeile 183 auf ein leeres Objekt angewendet wird